doc/release/upcoming_changes/23229.compatibility.rst

```diff
@@ -0,0 +1,2 @@
+- The ``busday_count`` method now correctly handles cases where the ``begindates`` and later in time
```

Comment:
Could you rephrase or add to say what "correctly" means here?  Something like: "previously the ``enddates`` was included when it should always be excluded." maybe?

"and later" -> is later or so.

Reply:
Done